plugins/admission/cmd/admission/main.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	polarisconfiguration "github.com/fairwindsops/polaris/pkg/config"
+	"github.com/sirupsen/logrus"
+	k8sConfig "sigs.k8s.io/controller-runtime/pkg/client/config"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+
+	fadmission "github.com/fairwindsops/insights-plugins/admission/pkg/admission"
+	"github.com/fairwindsops/insights-plugins/admission/pkg/models"
+)
+
+func exitWithError(message string, err error) {
+	if err != nil {
+		logrus.Fatalf("%s: %s", message, err.Error())
+	} else {
+		logrus.Fatal(message)
+	}
+}
+
+var handler fadmission.Validator
+
+var organization string
+var hostname string
+var cluster string
+
+func refreshConfig() error {
+	url := fmt.Sprintf("%s/v0/organizations/%s/clusters/%s/data/admission/configuration", hostname, organization, cluster)
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return err
+	}
+	token := strings.TrimSpace(os.Getenv("FAIRWINDS_TOKEN"))
+	req.Header.Set("Authorization", "Bearer "+token)
+	client := &http.Client{}
+	resp, err := client.Do(req)
+	if err != nil {
+		return err
+	}
+	if resp.StatusCode != http.StatusOK {
+		return fmt.Errorf("invalid status code: %d", resp.StatusCode)
+	}
+	defer resp.Body.Close()
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return err
+	}
+	var tempConfig models.Configuration
+	err = json.Unmarshal(body, &tempConfig)
+	if err != nil {
+		return err
+	}
+	if tempConfig.Polaris == nil {
+		polarisConfig, err := polarisconfiguration.ParseFile("")
+		tempConfig.Polaris = &polarisConfig
```

Comment:
should probably move this below the error check

Reply:
Oh dang! Yeah.